api/src/main/resources/messages.properties

```diff
@@ -245,4 +245,32 @@ isanteplusreports.number_of_suspected_cases_of_malaria_tested=Number of suspecte
 isanteplusreports.number_of_suspected_cases_of_confirmed_malaria=Number of suspected cases of confirmed malaria
 isanteplusreports.number_of_confirmed_cases_for_severe_malaria_and_put_on_treatment_according_to_the_national_standards_of_the_mspp=Number of confirmed cases for severe malaria and put on treatment according to the national standards of the MSPP
 isanteplusreports.number_of_patients_with_malaria_hospitalized=Number of patients with Malaria hospitalized
-isanteplusreports.number_of_cases_of_confirmed_malaria_for_pregnant_women=Number of cases of confirmed malaria for pregnant women
\ No newline at end of file
+isanteplusreports.number_of_cases_of_confirmed_malaria_for_pregnant_women=Number of cases of confirmed malaria for pregnant women
+
+isanteplusreports.pnls.label=ART Monthly Report
+isanteplusreports.pnls.result.department.label=Dep't
+isanteplusreports.pnls.result.clinic.label=Clinic
+isanteplusreports.pnls.result.Males.label=M
+isanteplusreports.pnls.result.FeMales.label=F
+isanteplusreports.pnls.result.Subtotal.label=SubTotal
+isanteplusreports.pnls.result.BreastFeeding.label=Breast Feeding
+isanteplusreports.pnls.result.1-0.label=0-1 Yrs
+isanteplusreports.pnls.result.1-4.label=1-4 Yrs
+isanteplusreports.pnls.result.5-9.label=5-9 Yrs
+isanteplusreports.pnls.result.10-14.label=10-14 Yrs
+isanteplusreports.pnls.result.15-19.label=15-19 Yrs
+isanteplusreports.pnls.result.20-24.label=20-24 Yrs
+isanteplusreports.pnls.result.25-29.label=25-29 Yrs
+isanteplusreports.pnls.result.30-34.label=30-34 Yrs
+isanteplusreports.pnls.result.35-39.label=35-39 Yrs
+isanteplusreports.pnls.result.40-44.label=40-44 Yrs
+isanteplusreports.pnls.result.45-49.label=45-49 Yrs
+isanteplusreports.pnls.result.50.label=50+ Yrs
+isanteplusreports.pnls.result.UnKnown.label=UnKnown Yrs
+isanteplusreports.pnls.result.Total.label=Total
+isanteplusreports.pnls.indicator.adult.label=ART Monthly Report Indicators
+isanteplusreports.pnls.result.pdf.header.label=ART Monthly Report
+isanteplusreports.pnls.result.pdf.creationDate.label=Date Created
+isanteplusreports.newlyEnrolledPatientsOnArt=NEW 1.1: Number of HIV+ newly enrolled on ART during the selected period (TX_NEW)
+isanteplusreports.referredInPatientsAnArt=REFERENCE previously enrolled People on ARVs and referees during the month
+isanteplusreports.newlyEnrolledBreastFeedingWomenOnArt=New breastfeeding women enrolled on ARVs
```

Comment:
For ease of validation and consistency, please include the indicator number before the translation.